src/TakeoffController.cpp

```diff
@@ -251,8 +278,9 @@ bool TakeoffController::waitUntilReady()
     }
 
     // This time is just used to calculate any ramping that needs to be done.
-    last_update_time_ = std::max(landing_detected_message_.header.stamp,
-                                 battery_interpolator_.getLastUpdateTime());
+    last_update_time_ = std::max({transform.header.stamp,
+                                  landing_detected_message_.header.stamp,
+                                  battery_interpolator_.getLastUpdateTime()});
```

Comment:
Is this valid syntax? just not sure, given the extra `{}` which was not used before.

Reply:
Aaron actually wrote this, it is an Initialization list. http://en.cppreference.com/w/cpp/utility/initializer_list